packages/xo-server/src/xo-mixins/backups-ng/index.mjs

```diff
@@ -533,4 +533,43 @@ export default class BackupNg {
 
     return backupsByVmByRemote
   }
+
+  async checkVmBackupNg(id, srId, settings) {
+    const restoredId = await this.importVmBackupNg(id, srId, settings)
+    const app = this._app
+    const xapi = app.getXapi(srId)
+    const restoredVm = xapi.getObject(restoredId)
+
+    try {
+      // remove vifs
+      await Promise.all(restoredVm.$VIFs.map(vif => xapi._deleteVif(vif)))
+
+      // start Vm
+      // await is handled by waitForGuestMetrics to control timeout
+      xapi.startVm(restoredId)
```

Comment:
Never let a promise dandling: a unhandled rejection can crash the process.

In this case, I think we can `await` it, IIRC, XAPI does not wait for the VM to be fully up before returning.

> FYI: in other cases where we really don't want to wait, we can ignore any errors `.catch(noop)`.

If you really don't want t